inlong-sort-standalone/pom.xml

```diff
@@ -113,6 +116,71 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+		<dependency>
```

Comment:
the dependency part should keep the same format.

Reply:
fixed